includes/class-wc-payments.php

```diff
@@ -831,19 +834,20 @@ public static function ajax_init_platform_checkout() {
 
 		$account_id = self::get_account_service()->get_stripe_account_id();
 
-		$platform_checkout_host = defined( 'PLATFORM_CHECKOUT_HOST' ) ? PLATFORM_CHECKOUT_HOST : 'http://host.docker.internal:8090';
+		$platform_checkout_host = defined( 'PLATFORM_CHECKOUT_FRONTEND_HOST' ) ? PLATFORM_CHECKOUT_FRONTEND_HOST : 'http://host.docker.internal:8090';
```

Comment:
Sure thing! I can do this prior to merging.

Actually, the reason I updated this was because in the codebase there is another reference to `[PLATFORM_CHECKOUT_FRONTEND_HOST](https://github.com/Automattic/woocommerce-payments/blob/685b09c72e7487299de8c5fd1ff249b022de6f61/includes/class-wc-payments-blocks-payment-method.php#L74)` while this line here uses `PLATFORM_CHECKOUT_HOST`. 

I am pretty sure these are meant to be the same, but can you confirm this is the case before we proceed?

Reply:
Looking at the git blame - @waclawjacek do you have any insight on the above? ☝️

Reply:
Ah. Makes sense. Thanks for clarifying. 👍